flows/nodes/runqueryadvanced.md

```diff
@@ -0,0 +1,33 @@
+# Run Query - Advanced Node
+
+This node executes a Gravwell query, much like the [Run a Query](runquery) node, but provides more advanced options for specifying the search duration. It outputs a structure into the payload (named `search` by default) which contains information about the search and allows other nodes to access the results.
+
+## Configuration
+
+* `Query String`, required: the Gravwell query to run.
+* `Start Time`, required: the starting point for the query timeframe. This can be either a literal timestamp ("2023-01-01T00:00:00Z07:00") or a [Go-style duration string](https://pkg.go.dev/time#ParseDuration) such as "-10h" or "-20m".
+* `End Time`, required: the end point for the query timeframe. This can be either a literal timestamp ("2023-01-31T11:59:59Z07:00") or a [Go-style duration string](https://pkg.go.dev/time#ParseDuration) such as "0h" or "-5h".
```

Comment:
Really minor observation:
Start Time has examples for hour and minute durations. The End Time uses hour for both examples. Recommend "-5m" instead.